src/main/java/com/HuePlugin/Fireworks.java

```diff
@@ -38,8 +38,12 @@ public void run() {
 
             }
         }
+        
+		this.room.setState(State.builder().color(Color.of(config.defaultColorConfig())).on());
```

Comment:
You can not retrieve information from the config in this file in this way. Maybe when creating thread, pass this color as an argument.

Reply:
Yup, that's on me. Changed!